hydro_std/src/bench_client/mod.rs

```diff
@@ -70,11 +70,23 @@ where
     let client_tick = clients.tick();
 
     // Set up an initial set of payloads on the first tick
-    let start_this_tick = client_tick.optional_first_tick(q!(()));
+    let (new_virtual_client_complete_cycle, new_virtual_client) =
+        client_tick.cycle_with_initial(client_tick.singleton(q!(0u32)));
+    new_virtual_client_complete_cycle.complete_next_tick(
+        new_virtual_client
+            .clone()
+            .map(q!(|virtual_id| virtual_id + 1)),
```

Comment:
This would eventually overflow, right? Should it be `min(num_clients_per_node, virtual_id + 1)`